cxx-qt-gen/src/gen_cpp.rs

```diff
@@ -385,7 +385,7 @@ fn generate_invokables_cpp(
             // we are passing *this across for cpp objects in rust.
             header: format!(
                 "Q_INVOKABLE {return_ident} {ident}({parameter_types});",
-                ident = invokable.ident.cpp_ident.to_string(),
+                ident = invokable.ident.cpp_ident,
```

Comment:
Huh, I actually didn't know you could use named identifiers in format strings. That's pretty cool! :)